brain/openfang_adapter.py

```
@@ -41,6 +41,16 @@ def _detect_provider_info(base_url: str, model: str) -> dict:
     url_lower = base_url.lower()
     model_lower = model.lower()
 
+    # 已知的 OpenAI-compatible 代理/中转 — 必须走 proxy，跳过后续 model-name 启发式匹配
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)